### ISSUE TITLE
How to specify background/border color for `extend`?

### ISSUE THREAD

**Issue by Slapbox** [vseventer/sharp-cli#72]
>  sharp extend <top> <bottom> <left> <right>   Extends/pads the edges of the image with the provided background colour

I've tried a few ways but I can't figure out how to specify a colour. Advice would be appreciated. Thank you!

**Comment by Slapbox** [comment]
Whoops I didn't realize there was a `--help` for each command.

> sharp extend 10 20 10 10 rgba(0,0,0,0)

**Comment by Slapbox** [comment]
Actually that didn't work and neither did `sharp extend 20 20 20 20 --background rgba(0,0,255,1)`

**Comment by vseventer** [comment]
> Actually that didn't work and neither did `sharp extend 20 20 20 20 --background rgba(0,0,255,1)`
> 
> > unknown file attribute: 0

Be sure to quote the background value, otherwise it is interpreted differently by your terminal. Try `--background "rgba(0,0,0255,1)".

**Comment by Slapbox** [comment]
Thanks so much for your speedy reply @vseventer! Unfortunately I spent probably half an hour trying darn near every permutation and even spent quite a while studying the source code for clues. I tried double quotes and single quotes, zsh shell and bash shell, etc etc.

Neither of these attempts worked either:

`sharp extend --top 20 --right 20 --bottom 20 --left 20 --background "rgba(0,0,255,1)"`

Error:
> Not enough non-option arguments: got 0, need at least 4

`sharp extend 20 20 20 20 --background "rgba(0,0,255,1)"`

Error:
> Input buffer contains unsupported image format


Any other ideas? Is it possible this option isn't working as intended?
